src/Cli/OrgCommands.php

```diff
@@ -132,29 +121,165 @@ public function orgAnalyze($org, $options = ['as' => 'default', 'format' => 'tab
+            if (!empty($data['content'])) {
+                $content = base64_decode($data['content']);
+                $ownerSource = 'file';
+                $codeowners = static::filterGlobalCodeOwners($content);
+            }
+        } catch (\Exception $e) {
+        }
+
+        return static::inferOwners($api, $org, $repoName, $codeowners, $ownerSource);
+    }
+
+    /**
+     * Get right column number based on column title.
+     */
+    public function getColumnNumber($columnTitle, $headerRow)
+    {
+        foreach ($headerRow as $key => $value) {
+            if ($value == $columnTitle) {
+                return $key;
+            }
+        }
+        throw new \Exception("Column $columnTitle not found in given header row.");
+    }
+
+    /**
+     * @command org:update-projects-info
+     * @param $csv_file The path to csv file that contains projects information.
      */
-    protected static function getSupportLevel($readme_contents)
+    public function orgUpdateProjectsInfo($csv_file, $options = [
+        'as' => 'default',
+        'update-codeowners' => false,
+        'codeowners-only-api' => false,
+        'codeowners-only-guess' => false,
+        'update-support-level-badge' => false,
+        'branch-name' => 'project-update-info',
+        'commit-message' => 'Update project information.',
+        'pr-body' => '',
+    ])
     {
-        $support_level = null;
-        $lines = explode("\n", $readme_contents);
-        $badges = SupportLevel::getSupportLevelBadges();
-        foreach ($lines as $line) {
-            foreach ($badges as $key => $badge) {
-                // Get the badge text from the badge markup.
-                preg_match('/^\[\!\[([A-Za-z\s\d]+)\]\(https:\/\/img.shields.io/', $badge, $matches);
-                if (!empty($matches[1])) {
-                    if (strpos($line, $matches[1]) !== false) {
-                        $support_level = $key;
-                        break 2;
+        $api = $this->api($options['as']);
+        $updateCodeowners = $options['update-codeowners'];
+        $codeownersOnlyApi = $options['codeowners-only-api'];
+        $codeownersOnlyGuess = $options['codeowners-only-guess'];
+        $updateSupportLevelBadge = $options['update-support-level-badge'];
+        if (!$updateCodeowners && !$updateSupportLevelBadge) {
+            throw new \Exception("Either --update-codeowners or --update-support-level-badge must be specified.");
+        }
+        if (!$updateCodeowners && ($codeownersOnlyApi || $codeownersOnlyGuess)) {
+            throw new \Exception("--codeowners-only-api and --codeowners-only-guess can only be used with --update-codeowners.");
+        }
+        if ($codeownersOnlyGuess && $codeownersOnlyApi) {
+            throw new \Exception("--codeowners-only-api and --codeowners-only-guess can't be used together.");
+        }
+        $prTitle = '[UpdateTool - Project Information] Update project information.';
```

Comment:
It would be useful to be able to specify the title and/or the JIRA story ("[CMS-1234]"), so that folks will know where to go for more info when a PR is opened against their project.

Reply:
Definitely. Done. Added as an option and I kept this text as the default option.